readme.md

```
@@ -0,0 +1,41 @@
+- ```cd ~ ``` or just  ```cd ``` --> change directory to the home directory
+
+- ``` cd - ``` --> Go to the last working directory.
+
+- ``` cd ..``` --> change directory to one step back.
+
+- ``` cd ../..``` --> Change directory to 2 levels back.
+
+- ``` mkdir  directoryName``` --> to make a directory in a specific location
+
+Examples:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)